cmds/core/tee/tee.go

```diff
@@ -25,19 +25,13 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
-const name = "tee"
-
 var (
 	cat    = flag.BoolP("append", "a", false, "append the output to the files rather than rewriting them")
 	ignore = flag.BoolP("ignore-interrupts", "i", false, "ignore the SIGINT signal")
 )
 
-// handeFlags parses all the flags and sets variables accordingly
-func handleFlags() int {
-	flag.Parse()
-
+func run(stdin io.Reader, stdout io.Writer, stderr io.Writer, args []string) error {
```

Comment:
consider following the more recent practice of making an options struct
type options struct {
cat bool
ignore bool
}
and passing it in to run. Or, possibly, having run work like exec.Command
func Command(args ...string) command {
}
and
type command struct {
args []string
in io.Reader
out, err io.Writer
cat bool
ignore bool
}

and then implement command.Run()
This will make your function unchanged even as people add more complexity to tail, as well as make tests much simpler. Globals and tests don't fit well together anyway.

But using exec.Command as a template might be more useful. I realize we're got into the habit of creating run functions, but ... it starts to look a bit clumsy with all these arguments. 
Can the length of args be 0? seems so: 
"The tee utility copies standard input to standard output, making a copy in zero or more files.  The output is unbuffered."
Maybe it should be variadic then?
you can even then have a New function which returns a command with reasonable defaults, i.e. output set to os.Stdout and so on.

Reply:
In general I like it, I was under impression that code size is an issue after reading u-root paper. So I tried not change stuff util I see the bug. 

In my next commit I tried to implement you command suggestion. By this command struct or options we can run tests in parallel, probably it will not speedup CI, but anyway its nice to have.

Reply:
thanks, also, if you can change the label to awating reviewer, it helps me know when to look again :-)

I'll take a look now anyway.

If you see a way to do things better, you are always welcome to show us!

Reply:
Cool, I don't think I can change labels in I'm my PR. At best I can re-request review.